harness/determined/cli/workspace.py

```diff
@@ -384,6 +431,21 @@ def yaml_file_arg(val: str) -> Any:
                     *CHECKPOINT_STORAGE_WORKSPACE_ARGS,
                     *DEFAULT_POOL_ARGS,
                     cli.Arg("--json", action="store_true", help="print as JSON"),
+                    cli.Arg(
+                        "--cluster-name",
+                        type=str,
+                        help="cluster within which we create the \
+                        workspace-namespace binding",
+                    ),
+                    cli.Group(
+                        cli.Arg(
+                            "-n",
+                            "--namespace",
```

Comment:
can we make this consistent with the add bindings arguments? either make both `namespace` or make both `namespace_name`

Reply:
Done! Changed everywhere to use `namespace` instead of `namespaceName`/`namespace_name`. 
On that end, can we change the db too so that instead of `namespace_name` in our `workspace_namespace_bindings` table, we use `namespace`?